docs/steps/sendMessage.md

```diff
@@ -0,0 +1,22 @@
+# sendMessage() step
+
+This step sends a CI message to the message bus. For more information, check the [CI Messages](https://pagure.io/fedora-ci/messages) project.
+
+**Warning** This step will be probably rapidly evolving in the near future, so the API may change.
```

Comment:
Not sure about this sentence. "probably", "rapidly", "near future"

So will it be evolving or not? :)